Source/Modules/go.cxx

```diff
@@ -10,6 +10,7 @@
 #include "swigmod.h"
 #include "cparse.h"
 #include <ctype.h>
+#include <stdint.h>
```

Comment:
what about I rewrite the siphash to use unsigned long (assumed to be 32-bit or larger) only?